src/node/transaction.cpp

```diff
@@ -13,10 +13,11 @@
 
 #include <future>
 
-TransactionError BroadcastTransaction(const CTransactionRef tx, uint256& hashTx, std::string& err_string, const CAmount& highfee)
+TransactionError BroadcastTransaction(const CTransactionRef tx, uint256& hashTx, std::string& err_string, const CAmount& max_tx_fee, bool relay, bool wait_callback)
```

Comment:
You could remove the `hashTx` parameter from this interface method. The only caller of the method supplies an unused uint256 and then throws it away.